app/controllers/api/configuration_script_sources_controller.rb

```diff
@@ -1,4 +1,26 @@
 module Api
   class ConfigurationScriptSourcesController < BaseController
+    def create_resource(_type, _id, data)
+      attrs = validate_attrs(data)
+      type = ConfigurationScriptSource.class_from_request_data(attrs)
+      raise 'type not currently supported' unless type.respond_to?(:create_in_provider_queue)
```

Comment:
as for the message, maybe something like "ConfigurationScriptSources cannot be added to #{manager_ident(manager)}"